src/lib/filter/adjudicators.ts

```diff
@@ -216,6 +216,22 @@ export const uncarryableNamespace = allPass([
   pipe((namespaceSelector, kubernetesObject) => namespaceSelector.includes(carriedNamespace(kubernetesObject)), not),
 ]);
 
+export const missingCarriableNamespace = allPass([
```

Comment:
What's it like if we try to add typing to this function for the inputs and the return type?